app/templates/src/unplanned-maintenance.html

```diff
@@ -0,0 +1,167 @@
+
+            <div class="header__container container">
+
+              <div class="logo header__logo">
+                <img src="/static/images/logo.svg" alt="Office for National Statistics" class="logo__img">
+              </div>
+
+            </div>
+
+          </div>
+
+        </div>
+
+        <div class="bar" role="banner">
+          <div class="bar__inner container">
+            <div class="bar__title venus">ONS Business Surveys</div>
+          </div>
+        </div>
+
+      </header>
+
+      <div class="page__subheader"></div>
+
+      <div class="container page__container">
+
+        <div class="grid">
+
+          <div class="grid__col col-6@m">
+            <div role="main" id="main" class="page__main">
+            <section>
+                <h1 class="saturn">Unplanned downtime</h1>
+                <div class="panel panel--simple panel--error panel--spacious">
+                <p>We are currently experiencing technical difficulties. Our engineers are investigating and hope to have the service back up shortly.
+                  <p>Thank you for your patience.
+                </div>
+
+            </section>
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </div>
+    <footer class="footer page__footer">
+      <div class="footer-nav container" role="contentinfo">
+
+        <div class="grid container">
+          <div class="">
+
+            <div class="grid__col col-4@m">
+              <div class="footer-nav__heading">
+                <h3 class="venus">Legal information</h3>
+              </div>
+              <ul class="footer-nav list">
+                <!--
+                <li class="footer-nav__item">
+                  <a href="" class="footer__link">Cookies and privacy</a>
+                </li>
+              -->
+                <li class="footer-nav__item">
+                  <a href="https://www.ons.gov.uk/help/termsandconditions" class="footer__link">Terms and conditions</a>
+                </li>
+              </ul>
+            </div>
+
+            <div class="grid__col col-4@m">
+              <div class="footer-nav__heading">
+                <h3 class="venus">About ONS</h3>
+              </div>
+              <ul class="footer-nav list">
+                <li class="footer-nav__item">
+                  <a href="https://www.ons.gov.uk/aboutus/whatwedo" class="footer__link">What we do</a>
+                </li>
+                <!--
+                <li class="footer-nav__item">
+                  <a href="" class="footer__link">Contact us</a>
+                </li>
+                -->
+                <li class="footer-nav__item">
+                  <a href="https://www.ons.gov.uk/help/accessibility" class="footer__link">Accessibility</a>
+                </li>
+              </ul>
+            </div>
+
+            <div class="grid__col col-4@m">
+              <div class="footer-nav__heading">
+                <h3 class="venus">Statistics</h3>
```

Comment:
Seeing the planet names reminds me that we should probably use the design system for this so we can keep all our css in line.  Probably out of scope for this one as we just need to get something that works in place first, then we can improve upon it.

Reply:
If this does need to be done but is out of scope for this can we please raise a card for it to be done  so that we don't forget about it?

Reply:
I think this is out of the scope of this ticket, I'll make a card for it